blocks/single-column-layout-block/_index.scss

```diff
@@ -1,16 +1,23 @@
 @use "@wpmedia/arc-themes-components/scss";
 
-.b-single-column-regular-header {
-	@include scss.block-properties("single-column-regular-header");
-}
-
 .b-single-column-regular {
-	max-width: 90rem; // 1440px
-	width: 100%;
-	margin: auto;
-	@include scss.block-properties("single-column-regular");
-}
+	&__header {
+		@include scss.block-properties("single-column-regular-header");
+		@include scss.block-components("single-column-regular-header");
+	}
+
+	&__main {
+		max-width: 90rem; // 1440px
+		width: 100%;
```

Comment:
Should we move this into the theme CSS and use the new alias values or does it make sense to keep this here?

Reply:
I wasn't sure what was reasonable here for keeping. I can move this to the theme file if that's the recommendation

Reply:
updated to move the scss to the token file

Reply:
I don't think we should move this to the token file - we can make an assumption that the width is always 100% - a client could set a max-width to control the maximum width

Reply:
ok will keep the max-width in the token file. will keep the width 100% in the layout. right?

Reply:
updated